support-frontend/assets/helpers/redux/checkout/product/actions.ts

```diff
@@ -9,7 +9,6 @@ export const {
 	setProductPrices,
 	setAllAmounts,
 	setSelectedAmount,
-	setCoverTransactionCost,
```

Comment:
Could this come out of the corresponding `reducer.ts` file?

Reply:
It could, but I figured it was a marginal gain to remove it and it will come out when we delete most of the redux